packages/pn-pa-webapp/cypress/e2e/NewNotificationAdministratorWithGroups.cy.ts

```diff
@@ -0,0 +1,82 @@
+import { cesare } from '../fixtures/recipients/recipients';
+import {CREATE_NOTIFICATION} from '../../src/api/notifications/notifications.routes';
+import { PNRole } from '../../src/models/user';
+import { User } from '../../src/redux/auth/types';
+describe("Notifications New Notification", () => {
+  const pdfTest1 = './cypress/fixtures/attachments/pdf_test_1.pdf';
+  const pdfTest2 = './cypress/fixtures/attachments/pdf_test_2.pdf';
+
+  beforeEach(() => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      return false;
+    });
+    cy.viewport(1920, 1080);
+
+    cy.logout();
+    cy.loginWithTokenExchange();    
+
+    // intercepts send notification request stubbing its successful response
+    cy.intercept('POST', CREATE_NOTIFICATION(), {
+      statusCode: 202,
+      body: {
+        notificationRequestId: 'S0FOQS1VV1JBLUFLSkUtMjAyMjExLVAtMQ==',
+        paProtocolNumber: 'prot-12555514'
+      },
+    }).as('saveNewNotification');
+    
+    cy.intercept('/delivery/attachments/preload').as('preloadAttachments');
+    
+    cy.visit('/dashboard/nuova-notifica');
+  });
+
+  it.only('Create new notification with groups and administrator role', () => {
+
+    cy.window().then((win) => {
```

Comment:
Move the logic of accociating a role to the current user inside a new login command like cy.loginAsAdmin() and then refer to it instead of cy.loginWithTokenExchange()